examples/prediction/xgboost-high-level-apis-azure.ipynb

```diff
@@ -0,0 +1,349 @@
+{
```

Comment:
Is that better to update the current notebook file to add a cell for Azure backends? since there are some duplicated code and if each backend introduce one, that's not good for maintain.

Reply:
@jinchihe This was actually my question at https://github.com/kubeflow/fairing/pull/322#issuecomment-513196543 and I would personally prefer a single notebook. This is also one reason I marked as WIP, but if the consensus is a single notebook, I am happy to change the implementation.

Reply:
It makes sense to have at least have one single notebook that demonstrates all backends. Apart from that we may create notebooks highlighting specific backends if needed.

My only concern is that if any of the integration tests are exercising this notebook we need to ensure that the single notebook with all backends is able to execute periodically as part of the integration tests.